templates/vars.yml

```diff
@@ -15,3 +15,8 @@ versions:
   java:   ['8', '11']
   hadoop: ['3.3.2']
   scala:  ['2.12', '2.13']
+
+- spark:  ['3.3.1']
+  java:   ['8', '11']
+  hadoop: ['3.3.2']
+  scala:  ['2.12', '2.13']
```

Comment:
LGTM, though you could have merged with above to do since all components other than `spark` are the same:

```
- spark: ['3.3.0', '3.3.1']
  ...
```